lib/iso_spinner.py

```diff
@@ -0,0 +1,128 @@
+# Copyright (C) IBM Corp. 2016.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import glob
+import logging
+import os
+import re
+
+from lib import build_system
+from lib import exception
+from lib import utils
+from lib import distro_utils
+
+LOG = logging.getLogger(__name__)
+
+
+class MockPungiSpinner(object):
+
+    def __init__(self, config):
+        self.config = config
+        (self.distro, self.version, self.arch) = distro_utils.detect_distribution()
+        # only the major number must be consider
+        self.version = re.sub("\..*$", "", self.version)
+
+    def _run_mock_command(self, cmd):
```

Comment:
It was decided to keep rpm package builder and ISO image creator codes in separate modules, since they do not share any code.

Just a quick note: Mock class, in my opinion, is a misleading name because it is not a wrapper for mock command as you may notice, it is just rpm packages builder that uses mock.